### ISSUE TITLE
Create new user 

### ISSUE THREAD

**Issue by robibrazze** [3liz/lizmap-web-client#3681]
### What is the question?

Bye,
I cannot set up a new user in the lizmap v.3.5 administration panel.
First it creates a group called pippo with identifier pippo1, then I create new user called test_pippo and in the mail field I put my mail...but it gives error.
Is there no way to avoid sending mail to create new user pw poppo1 without touching localconfig.ini.php?

thanks

### Versions

Lizmap web client 3.5

### Check Lizmap plugin

- [X] I have done the step just before in the Lizmap QGIS desktop plugin before opening this ticket. Otherwise, my ticket is not considered valid and might get closed.

### Operating system

Ubuntu 20.04

### Browsers

Chrome

### Browsers version

Chrome

**Comment by Gustry** [comment]
Your question is not very clear.

> but it gives error.

What is the error ? What do you mean by "touching localconfig.ini.php?" 

Please follow the ticket template : QGIS plugin step to know more precisely your versions used, give your logs of Lizmap Web Client, provide more describe steps.

**Comment by robibrazze** [comment]
Hello @Gustry ,
you're right, I'll make it more precise below.
Currently, in the admin panel of LWC 3.5, i have only one admin user with a password that I do not remember setting up and I did not receive it via email. Now, I want to create a new user named "pippo_comune". I have created a group named "pippo" with an identifier of "users_pippo". However, I am unsure how to set up the password for the new user. When I tried to use my email address, I received an error message saying "Error: There was an error sending the email. Please try again later or contact the administrator." 

In summary, I would like to create a new user and set up their username and password. Can you please provide me with instructions on how to do this? Thank you for your help.

**Comment by katagen** [comment]
If you want to set up the password, I think you need to touch localconfig.ini.php
It is written in the documentation:

_By default, in Lizmap, when a new user is created, the user will get an email for setting up his own password. If for some reasons, the administrator needs to setup the password, the localconfig.ini.php needs to be edited:_

[jcommunity]
resetAdminPasswordEnabled = off

**Comment by Gustry** [comment]
Thanks @katagen
Indeed, it's in the documentation : https://docs.lizmap.com/current/en/admin/users-groups.html#setting-password-for-users

It's a best practice to not set the password for your user and to let them set it by themselves, but it's still possible to disable this.

For the mail, you should check your SMTP settings. Check your Lizmap logs.
